### PR TITLE
Equations

### PR DESCRIPTION
**Pull request by lbotti319**

### CODE REVIEW

File: equations.tex [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	rQ(t) - zQ(t) & Q(t) \geq q_{\text{max}}
+\end{cases}$
+
+$\frac{dZ}{dt} = zE(t) -kS(t)Z(t)$
+
+$\frac{dD}{dt} = kS(t)Z(t) + mS(t) + zQ(t) + C(t)E$
+
+$\frac{dX_S}{dt} = k_S X_S(t) (1 - X_S(t))[Z(t) + Q(t) - \epsilon_S L_S(t)]$
+
+$\frac{dX_I}{dt} = \begin{cases}
```
Comment (ghost) [inline equations.tex]: change x_I to x_E for consistency with our variables

File: equations.tex [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	k_I X_I(t) (1 - X_I(t))\{(z-r)[E(t) + Q(t)] - \epsilon_I\} & Q(t) < q_{\text{max}} \\
```
Comment (ghost) [inline equations.tex]: Change k_I to 'k'

File: equations.tex [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+$\frac{dQ}{dt} = \begin{cases}
+	(C_0 + X_I)E - rQ(t) - zQ(t) & Q(t) < q_{\text{max}}\\
+	rQ(t) - zQ(t) & Q(t) \geq q_{\text{max}}
+\end{cases}$
+
+$\frac{dZ}{dt} = zE(t) -kS(t)Z(t)$
+
+$\frac{dD}{dt} = kS(t)Z(t) + mS(t) + zQ(t) + C(t)E$
+
+$\frac{dX_S}{dt} = k_S X_S(t) (1 - X_S(t))[Z(t) + Q(t) - \epsilon_S L_S(t)]$
```
Comment (ghost) [inline equations.tex]: Since our k is constant throughout, change k_S to just 'k' (unless I'm missing something here)

File: equations.tex [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+$\frac{dX_I}{dt} = \begin{cases}
+	k_I X_I(t) (1 - X_I(t))\{(z-r)[E(t) + Q(t)] - \epsilon_I\} & Q(t) < q_{\text{max}} \\
+	0 & Q(t) \geq q_{\text{max}}
+\end{cases}$
+
+$\frac{dL_S}{dt} = \alpha_S C(t) - \mu L_s(t)$
```
Comment (ghost) [inline equations.tex]: We should update variable name alpha_S so it's not tied to infection probability (or update alpha)

**Review by ghost — CHANGES_REQUESTED** [review]

File: equations.tex (hunk 3 above)
Comment (lbotti319) [reply equations.tex]: I think you are. We have three different `k`'s. We have `k` the kill rate. `k_S` the learning rate of the susceptibles for taking drastic measures, and `k_E` the learning rate of the exposed for self quarantine.

File: equations.tex (hunk 1 above)
Comment (lbotti319) [reply equations.tex]: Done

File: equations.tex (hunk 4 above)
Comment (lbotti319) [reply equations.tex]: Changed to beta

File: equations.tex (hunk 3 above)
Comment (ghost) [reply equations.tex]: Got you. I missed writing that in my notes. Thanks for clarification.

**Review by ghost — APPROVED** [review]

**Pull request merged**